src/data.re

```diff
@@ -223,7 +223,7 @@ module Tier = {
       amount: 10,
       description: {js|
 - Logo on the Website
-- 1 sponsoring package for ReactVienna or ReasonVienna meetup (450€)
+- 1 sponsoring package for the ReasonVienna meetup
```

Comment:
Why removing ReactVienna? I think it has much more value for loca companies.

Reply:
@nikgraf was not happy with this decision... I agree, it will make things more complex and also take away potential resources from ReactVienna

Reply:
@okonet If we wanna include ReactVienna there should be a formal agreement, because right now it's not clear how much money should be spent on a meetup then and how much on the conference.